app/src/shared/components/Button/newButton.pcss

```diff
@@ -0,0 +1,256 @@
+.root {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  padding: 0;
+  background-color: transparent;
+  border: 1px solid;
+  border-radius: 4px;
+  font: var(--sans);
+  font-weight: var(--medium);
+  letter-spacing: 0;
+  text-align: center;
+  transition: 200ms ease-in-out;
+  transition-property: border-color, background-color, color, transform;
+  white-space: nowrap;
+  cursor: pointer;
+  user-select: none;
+
+  &,
+  &:--idle,
+  &:--enter {
+    text-decoration: none;
+    outline: none;
+  }
+
+  &:--enter {
+    transition-duration: 5ms;
+    transform: scale(1.05);
+  }
+
+  &:active {
+    transform: scale(0.95);
+    outline: none;
+  }
+
+  &[disabled] {
+    opacity: 0.5;
+    cursor: not-allowed;
```

Comment:
Maybe let's enforce `scale(1)` on all disabled buttons? Otherwise hover (and focus?) will resize them altho they're grayed out.

Reply:
Maybe we could tell disabled buttons to ignore any interaction? `pointer-events: none;` + `tabIndex="-1"`? Thinking out loud.

Here, in storybook, a hovered disabled button scales up and gets darker.

<img width="398" alt="Screenshot 2019-11-13 11 06 52" src="https://user-images.githubusercontent.com/320066/68753777-dec0c580-0605-11ea-82b3-b5a0dc830653.png">